maintenance/2020-09-22-infra-compute-upgrade.md

```diff
@@ -0,0 +1,5 @@
+## Compute cluster upgrade
+
+On Tue, 2020-09-22, compute cluster will be upgraded.
+During the maintenance, our infrastructure team will upgrade only control-plane components.
+Performance of some [foundries.io](https://foundries.io/foundries.io) services might be impacted, but service interruption is not expected.
```

Comment:
Just a minor point: the url doesn't seem to be correct.
```suggestion
Performance of some [foundries.io](https://foundries.io) services might be impacted, but service interruption is not expected.
```

Reply:
Nice catch!